test/tests/wasm/example.wast

```diff
@@ -0,0 +1,7 @@
+(module
+  (func $exampleImport (import "example" "exampleImport") (param i32) (result i32))
+  (func $exampleExport (export "exampleExport") (param $value i32) (result i32)
+    get_local $value
+    call $exampleImport
+  )
+)
```

Comment:
this imports `example` module, linking `exampleImport` function. It exports a single function `exampleExport` which when called with an integer will then itself call `exampleImport`, passing along its return value so we can confirm indeed everything was linked correctly